app/settings.py

```diff
@@ -30,3 +35,9 @@ def get_key(key_name):
 FTP_FOLDER = os.getenv('SEFT_CONSUMER_FTP_FOLDER', '.')
 
 SDX_KEYS_FILE = os.getenv('SDX_KEYS_FILE', 'keys.yml')
+
+# Configure the number of retries attempted before failing call
+session = requests.Session()
```

Comment:
I think the total retries and backoff_factor should be variables in this file.

However the session object itself should be created in the main.py file